src/core/client/admin/components/UserRole/SiteModeratorActionsModal.tsx

```diff
@@ -70,68 +72,53 @@ const SiteModeratorActionsModal: FunctionComponent<Props> = ({
               <form onSubmit={handleSubmit}>
                 <HorizontalGutter spacing={3}>
-                  {mode === "promote" ? (
-                    <Localized
-                      id="community-assignYourSitesTo"
-                      elems={{ strong: <ModalHeaderUsername /> }}
-                      vars={{ username: username || notAvailableTranslation }}
-                    >
-                      <ModalHeader>
-                        Assign your sites to{" "}
-                        <ModalHeaderUsername>{username}</ModalHeaderUsername>
-                      </ModalHeader>
-                    </Localized>
-                  ) : (
-                    <Localized id="community-removeSiteModeratorPermissions">
-                      <ModalHeader>
-                        Remove Site Moderator permissions
-                      </ModalHeader>
-                    </Localized>
-                  )}
+                  <Localized
+                    id="community-assignYourSitesTo"
+                    elems={{ strong: <ModalHeaderUsername /> }}
+                    vars={{ username: username || notAvailableTranslation }}
+                  >
+                    <ModalHeader>
+                      Manage site permissions for{" "}
+                      <ModalHeaderUsername>{username}</ModalHeaderUsername>
+                    </ModalHeader>
+                  </Localized>
                   {submitError && (
                     <CallOut color="error" fullWidth>
                       {submitError}
                     </CallOut>
                   )}
-                  {mode === "promote" ? (
-                    <>
-                      <Localized id="community-siteModeratorsArePermitted">
-                        <ModalBodyText>
-                          Site moderators are permitted to make moderation
-                          decisions and issue suspensions on the sites they are
-                          assigned.
-                        </ModalBodyText>
-                      </Localized>
-                      <ModalBodyText>
-                        <Localized id="community-assignThisUser">
-                          <Typography variant="bodyCopyBold">
-                            Assign this user to
-                          </Typography>
-                        </Localized>
-                      </ModalBodyText>
-                    </>
-                  ) : (
-                    <Localized id="community-userNoLongerPermitted">
-                      <ModalBodyText>
-                        User will no longer be permitted to make moderation
-                        decisions or assign suspensions on:
-                      </ModalBodyText>
+                  <Localized id="community-siteModeratorsArePermitted">
+                    <ModalBodyText>
+                      Site moderators are permitted to make moderation
+                      suspensions on the sites they are assigned.
```

Comment:
Has this been updated in `admin.ftl`?

Reply:
oh i actually didnt mean to update this! changed it back